layers/meta-balena-jetson/conf/layer.conf

```diff
@@ -24,4 +24,5 @@ BBMASK += "/meta-tegra/recipes-graphics/ "
 BBMASK += "/meta-tegra/external/openembedded-layer/recipes-support/opencv/ "
 
 LAYERSERIES_COMPAT_balena-jetson = "sumo"
-
+KERNEL_ARGS_jetson-xavier = "console=ttyTHS0,115200 OS=l4t fbcon=map:0"
+SERIAL_CONSOLE_jetson-xavier = "115200 ttyTHS0"
```

Comment:
SERIAL_CONSOLE has been deprecated in favor of "SERIAL_CONSOLES". So this should be

> SERIAL_CONSOLES = "115200;ttyTHS0"